kubernetes-client-openapi-watcher/build.gradle

```diff
@@ -0,0 +1,49 @@
+import io.micronaut.kubernetes.client.openapi.tasks.CreateWatcherSpec
+
+plugins {
+    id 'io.micronaut.build.internal.kubernetes-module'
+    id 'io.micronaut.openapi' version libs.versions.micronaut.gradle.plugin
+    id 'io.micronaut.kubernetes.client.openapi'
+}
+
+micronautBuild {
+    binaryCompatibility {
+        enabled.set(false)
```

Comment:
:x: Instead of disabling, you can use `enabledAfter("...")`

Reply:
Modified